front/components/assistant/conversation/input_bar/editor/useHandleMentions.tsx

```diff
@@ -15,20 +15,8 @@ const useHandleMentions = (
 ) => {
   const stickyMentionsTextContent = useRef<string | null>(null);
 
-  // Memoize the mentioned agents to avoid unnecessary recalculations.
-  const mentionedAgentConfigurationIds = useMemo(() => {
-    let mentions: AgentMention[] = [];
-    if (stickyMentions?.length) {
-      mentions = stickyMentions;
-    } else if (selectedAssistant) {
-      mentions = [selectedAssistant];
-    }
-
-    return Array.from(new Set(mentions.map((m) => m.configurationId)));
-  }, [stickyMentions, selectedAssistant]);
-
   useEffect(() => {
-    if (mentionedAgentConfigurationIds.length === 0) {
+    if (!stickyMentions || stickyMentions.length === 0) {
```

Comment:
This does not account anymore for the "selectedAssistant" usecase.

Reply:
I thought we had discussed IRL :sweat_smile: no need to account for it, the selectedAssistant is separated from stickymentions and repurposed for new.tsx with the useEffect below in the pr